### ISSUE TITLE
Will this work client-side?

### ISSUE THREAD

**Issue by abalter** [alanshaw/markdown-pdf#141]
Is it possible to use this purely client-side?

**Comment by anko** [comment]
No. Markdown-pdf requires [phantomjs](https://github.com/ariya/phantomjs), which is a separate program.

However, if you construct a web page with [appropriate print CSS](https://www.smashingmagazine.com/2011/11/how-to-set-up-a-print-style-sheet/), you could just print to PDF through your web browser's usual print dialog.
